schema.sql

```diff
@@ -9,4 +9,26 @@ CREATE TABLE animal (
     weight_kg decimal
 );
 
-Alter TABLE animal ADD species char(20);
\ No newline at end of file
+Alter TABLE animal ADD species char(20);
+
+
+CREATE TABLE owners (
+    id int GENERATED BY DEFAULT AS IDENTITY,
+    full_name varchar(100),
+    age decimal,
+    PRIMARY KEY (id)
+);
+
+
+CREATE TABLE species (
+    species_id int GENERATED BY DEFAULT AS IDENTITY,
+     name varchar(100),
+    age int,
+    PRIMARY KEY (species_id)
+);
+
+
+ALTER TABLE animal ADD PRIMARY KEY (id);
+ALTER TABLE animal DROP COLUMN species;
+ALTER TABLE animal ADD COLUMN owner_id INT REFERENCES owners(id);
+ALTER TABLE animal ADD COLUMN species_id INT REFERENCES species(species_id);
```

Comment:
- Please, kindly ensure your animal table is pluralized as `animals` because the table has many animals. please kindly update this.